activation/handler.go

```diff
@@ -139,7 +139,7 @@ func (h *Handler) ProcessAtx(ctx context.Context, atx *types.ActivationTx) error
 // SyntacticallyValidateAtx ensures the following conditions apply, otherwise it returns an error.
 //
 //   - If the sequence number is non-zero: PrevATX points to a syntactically valid ATX whose sequence number is one less
-//     than the current ATX's sequence number.
+//     than the current ATXs sequence number.
```

Comment:
i don't think SequenceNumber used anymore. since you are here. can you update the comment so it make sense to you?

Reply:
PR can be found here: https://github.com/spacemeshos/go-spacemesh/pull/3520

Reply:
oh. wait. i thought there was a field called SequenceNumber (as the comment indicated). 
but i don't know about the field Sequence tho. it may be still useful. i don't know. sorry for the confusion.